### ISSUE TITLE
[🐞] useVisibleTask modules are not refreshing in 1.6.0

### ISSUE THREAD

**Issue by manucorporat** [QwikDev/qwik#6634]
### Which component is affected?

Qwik Rollup / Vite plugin

### Describe the bug

In the latest release 1.6.0, changes to `useVisibleTask$` are not refreshing, no HMR. I requires to restart the server.

### Reproduction

https://stackblitz.com/edit/qwik-starter-kr1cpt?file=src%2Froutes%2Findex.tsx,package.json

### Steps to reproduce

Change the body of `useVisibleTask`, changes are not reflected on HMR, a full server restart is needed.

### System Info

```shell
Command does not work.
```

### Additional Information

In development the same issue also causes:
```
Failed to fetch dynamically imported module: 
```
and 
```
Missing Qwik City Env Data
```
which might be related to the latest issues reported in https://github.com/QwikDev/qwik/issues/6237

**Comment by frankroc2022** [comment]
I tried very old qwik versions and also older vite versions. JS changes are not visible anymore, I think it's a general browser problem. (HTML and CSS changes are visible).
The following problems I don't have in a fresh 1.6.0. install (and even can't reproduce them, tried useVisibleTask, UseOnDocument and get data from another server):
Failed to fetch dynamically imported module: 
Missing Qwik City Env Data

**Comment by DustinJSilk** [comment]
Downgrading to 1.5.7 works for now.

**Comment by devcaeg** [comment]
The error is also caused by "useOnDocument".

**Comment by frankroc2022** [comment]
I started my project with qwik 0.11.0. now it's 1.6.0.

Since I readded the row <link rel="manifest" href="/manifest.json" /> in root.tsx and restarted dev server useOnDocument is working again. (I deleted the row a long time ago because it made problems while developing with vite and I mean chrome. I also changed in root.tsx body lang=de" back to "en", but I don't think that matters. Now I have only the problem with my get data from another server. But in fresh 1.6.0 it works.

**Comment by wmertens** [comment]
I think I fixed it in #6629

**Comment by manucorporat** [comment]
Happy to give it a try, do we have a dev release?

**Comment by wmertens** [comment]
@manucorporat yes check out https://github.com/QwikDev/qwik/pull/6629#issuecomment-2201091318

**Comment by wmertens** [comment]
but this problem still exists https://github.com/QwikDev/qwik/issues/6237#issuecomment-2205966207

**Comment by shairez** [comment]
Fixed in #6629 

Thanks @wmertens !
